### PR TITLE
Last changes

### PR DESCRIPTION
**Pull request by alvarochapela**
PR del modulo 1 como me indicó Pablo

### CODE REVIEW

**Comment by pabaldonedo** [comment]
I made a mistake when doing the reverts to keep the main clean and creating this one. That's why you may see some commits now of my own. Howver, the code was kept exactly the same as you had it.

File: .DS_Store
Comment (pabaldonedo) [inline .DS_Store:1]: NEVER add these kind of files to git.

You can use the [`.gitignore`](https://git-scm.com/docs/gitignore) file to automatically ignore certain files

File: src/.DS_Store
Comment (pabaldonedo) [inline src/.DS_Store:1]: same comment as before

File: src/Homework1.py
Comment (pabaldonedo) [inline src/Homework1.py:1]: There are two files with the same name, this should not be the case. I'm reviewing the one inside module_1 folder

File: src/__init__.py
Comment (pabaldonedo) [inline src/__init__.py:1]: This is dummy content that should be removed

File: src/module_1/Homework1.py
Comment (pabaldonedo) [inline src/module_1/Homework1.py:1]: following naming conventions [PEP8](https://peps.python.org/pep-0008/) filenames should be lowercase

File: src/module_1/module_1_meteo_api.py
Comment (pabaldonedo) [inline src/module_1/module_1_meteo_api.py:1]: Either submit the solution here or remove this file

File: src/module_1/Homework1.py
Comment (pabaldonedo) [reply src/module_1/Homework1.py:1]: This is the teachers solutions. While it is good that you review it and try it yourself, the point of the PRs is that you get personalised feedback of your work so you can spot your weaknesses and learn

**Review by pabaldonedo — CHANGES_REQUESTED** [review]
This is the teachers solutions. While it is good that you review it and try it yourself, the point of the PRs is that you get personalised feedback of your work so you can spot your weaknesses and learn

For example, in the files that were submitted by you I have made some comments that may be useful for you in the future

**Review by alvarochapela — COMMENTED** [review]
Everything reviewed

File: src/__init__.py
Comment (pabaldonedo) [inline src/__init__.py:1]: The dummy content to be removed was the internals of the file, not te file in itself. If you are unfamiliar with __init__ files check [this](https://docs.python.org/3/reference/import.html#regular-packages). Please restore this file but with empty content

File: src/module_1/__init__.py
Comment (pabaldonedo) [inline src/module_1/__init__.py:1]: same here

**Review by pabaldonedo — CHANGES_REQUESTED** [review]

**Review by pabaldonedo — APPROVED** [review]

**Review by pabaldonedo — APPROVED** [review]

**Pull request merged**